tests/test_tutorial/test_first_steps/test_tutorial001.py

```diff
@@ -18,6 +18,9 @@ def test_get_path(path, expected_status, expected_response):
     assert response.status_code == expected_status
     assert response.json() == expected_response
 
+    response = client.head(path)
+    assert response.status_code == expected_status
+
```

Comment:
We also need to test:

a) Response to HEAD request does not contain body data
b) The response headers to HEAD request are the same as the headers to GET request

Reply:
> Response to HEAD request does not contain body data

We don't. This is tested on the underneath packages.

Reply:
Why do we need this?

This is the tutorial tests, and in tutorial tests we want to demonstrate basic functionalities to package users. This is sort of a supplement to documentation, and from my perspective, this would be a nice to have.